plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlab.ts

```diff
@@ -175,6 +185,10 @@ export function createPublishGitlabAction(options: {
         visibility: repoVisibility,
       });
 
+      if (topics.length) {
+        await client.Projects.edit(projectId, { topics });
```

Comment:
This can go in the `create` call, right? Should work to do something like

```ts
      const { id: projectId, http_url_to_repo } = await client.Projects.create({
        namespace_id: targetNamespace,
        name: repo,
        visibility: repoVisibility,
        ...(topics.length ? { topics } : {}),
      });
```

Reply:
Good catch, I'll update this asap.